### ISSUE TITLE
[JENKINS-38716] editor not accessible to screenreader

### ISSUE THREAD

**Issue by jenkins-infra-bot** [timja-org/jenkins-gh-issues-poc-11-23#6246]
<p>Although most/all of jenkins is usable by users using a screenreaders like me, it seems that the editor is not.<br/>
I mean at least some places where I can enter multiline data, and that is for example editor of configuration files when using config file plugin, editor where you can enter shell scripts to execute for a build step, or anything that you can enter when your markup formatter is switched from plain to any other.<br/>
Accessibility events are not reported when moving through the editor, so I do not know anything when I arrow through the text, there is just no feedback as it goes for normal html fields, so I cannot review what I type and read anything from the field, by any means.</p>

---
<details><summary><i>Originally reported by <a href="https://issues.jenkins.io/secure/ViewProfile.jspa?name=webczat">webczat</a>, imported from: <a class="no-jira-link-rewrite" href="https://issues.jenkins.io/browse/JENKINS-38716" target="_blank">editor not accessible to screenreader</a></i></summary>
<i><ul>
<li><b>status</b>: Resolved
<li><b>priority</b>: Minor
<li><b>component(s)</b>: core
<li><b>label(s)</b>: accessibility
<li><b>resolution</b>: Duplicate
<li><b>resolved</b>: 2016-10-19T08:43:47+00:00
<li><b>votes</b>: 0
<li><b>watchers</b>: 1
<li><b>imported</b>: 2025-11-23
</ul></i>
<details><summary>Raw content of original issue</summary>

<pre>
<p>Although most/all of jenkins is usable by users using a screenreaders like me, it seems that the editor is not.
I mean at least some places where I can enter multiline data, and that is for example editor of configuration files when using config file plugin, editor where you can enter shell scripts to execute for a build step, or anything that you can enter when your markup formatter is switched from plain to any other.
Accessibility events are not reported when moving through the editor, so I do not know anything when I arrow through the text, there is just no feedback as it goes for normal html fields, so I cannot review what I type and read anything from the field, by any means.</p></pre>
</details>
</details>
